src/main/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/FindOperation.java

```diff
@@ -527,6 +530,7 @@ private List<SimpleStatement> buildSortedSelectQueries(DBFilterBase.IDFilter add
                   .from(commandContext.namespace(), commandContext.collection())
                   .where(expression)
                   .limit(maxSortReadLimit())
+                  .allowFiltering(expressionBuiltResult.allowFiltering())
```

Comment:
Thought we are not going to do allow filtering., we will be using the map columns like `query_text_values` etc for !- on _id and won't support `$nin`.